guessInput.js

```diff
@@ -1,5 +1,88 @@
+//function changes values to uppercase
+let makeUpper = (values) => {
+    return values.toUpperCase()
+}
+
+//handles key presses in the guess box
 const guessHandler = () => {
-    console.log(guessInput.value)
+    let guessInputArray = [guessInput.value]
+    let upperGuessInput = guessInputArray.map(makeUpper)
+//filter function loops through user guesses and compares to counties list. Adds to new array if correct.
+    let correctInputArray = (upperGuessInput.filter((answer) => {
+        let match = false
```

Comment:
lines 12-22 can be replaced with:

`    let correctInputArray = (upperGuessInput.filter((answer) => {
        return upperCounties.includes(answer);
    }))